contracts/access/manager/AccessManager.sol

```diff
@@ -0,0 +1,498 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IAccessManager} from "./IAccessManager.sol";
+import {IManaged} from "./IManaged.sol";
+import "../../utils/Address.sol";
+import "../../utils/Context.sol";
+import "../../utils/types/Time.sol";
+
+
+
+
+
+
+
+contract DelayedActions is Context {
```

Comment:
`withDelay` and `_executeCheck` are not used. Overall I would merge this into `AccessManager`.

Reply:
For the record, `_executeCheck` is used in `AccessManager.relay`